<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MergeOptionsDialog</class>
 <widget class="QDialog" name="MergeOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Merge Operation</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin" stdset="0">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Merge Operation Setup</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin" stdset="0">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="7" column="1" colspan="3">
       <widget class="QComboBox" name="hier_mode_cbx">
        <item>
         <property name="text">
          <string>Flat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Top cell only</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Individually for current and subcells (semi hierarchical)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>(Layer is overwritten)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Layer</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="lay::CellViewSelectionComboBox" name="cvr_cbx"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="threshold_le"/>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Hierarchy</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="lay::CellViewSelectionComboBox" name="cv_cbx"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Result</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Source</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Layer</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="3">
       <widget class="QCheckBox" name="min_coherence_cb">
        <property name="text">
         <string>Minimum coherence (for touching corners)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="lay::LayerSelectionComboBox" name="layer_cbx">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="lay::LayerSelectionComboBox" name="layerr_cbx">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>(0: all polygons, 1: at least two overlapping ...)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Layout and cell</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Layout and cell</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Overlap threshold</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>506</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lay::LayerSelectionComboBox</class>
   <extends>QComboBox</extends>
   <header>layWidgets.h</header>
  </customwidget>
  <customwidget>
   <class>lay::CellViewSelectionComboBox</class>
   <extends>QComboBox</extends>
   <header>layWidgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MergeOptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MergeOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
